### PR TITLE
Fix time for Rawlh source

### PR DESCRIPTION
**Pull request by Pavkazzz**

### CODE REVIEW

**Comment by Pavkazzz** [comment]
My bad, fixed

File: src/ja/rawlh/build.gradle [hunk 1]
```diff
@@ -6,8 +6,8 @@ ext {
     pkgNameSuffix = "ja.rawlh"
     extClass = '.Rawlh'
     extVersionCode = 1
-    extVersionSuffix = 0
-    libVersion = '1.0'
+    extVersionSuffix = 1
+    libVersion = '1.1'
```
Comment (paronos) [inline src/ja/rawlh/build.gradle]:
```kotlin
extVersionCode = 2
extVersionSuffix = 1
libVersion = '1.0'
```

Comment (CarlosEsco) [reply src/ja/rawlh/build.gradle]: So I would do 2,0,1.2. 

1.2 just signifies what tachiyomi library version to use and 1.2 is the latest.

**Comment by Pavkazzz** [comment]
Done

**Pull request merged**